mne/channels/tests/test_montage.py

```diff
@@ -1205,6 +1205,83 @@ def test_set_montage_with_missing_coordinates():
     )
 
 
+def test_get_channel_coordinates():
+    """Test get montage and its coordinates."""
+    N_CHANNELS = 3
+
+    # create toy dataset
+    raw = _make_toy_raw(N_CHANNELS)
```

Comment:
make your test with testing data as you'll have more channel types. Having only EEG or iEEG is too easy.

doing

raw.get_ch_positions()

seems to miss certain EEG channels on testing data.

Reply:
I am now using 'test_raw.fif', 'test.vhdr', and 'sample_audvis_raw.fif' to do `get_montage()` -> `set_montage()`.

I added the unit tests for `get_ch_positions()`.